Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -2262,6 +2262,18 @@ public void TestConnectStringWithQueryTag()
             }
         }
 
+
+        [Test]
+        [IgnoreOnCI("This test requires a valid connection string in the configuration file.")]
+        public void TestLocalDefaultConnectStringReadFromToml()
+        {
+            using (var conn = new SnowflakeDbConnection())
+            {
+                conn.Open();
```

Comment:
I know, but how to execute this test locally? It requires manual work to prepare a toml file.
If the test is not automated it is not likely anyone would execute it in future.
We could write preparing such a toml file. Otherwise no one would execute the test in the future.

Reply:
It requires manual work from the user to run this test; the toml feature currently will look up the toml location based on the SNOWFLAKE_HOME env variable or the default by the operating system. If we have a toml file, maybe created from the parameters.json, it will require overriding the SNOWFLAKE_HOME to point to that location but could generate undesired behaviors if something fails. This test will run if the users runs the SFConnectionIT tests and fails if the environment is not prepared locally. Maybe I could add the steps as a comment in the test to run it locally. wdyt? @sfc-gh-knozderko

Reply:
Added test with scenarios with toml file automatic generation based on ConnectionString

Reply:
https://github.com/snowflakedb/snowflake-connector-net/blob/d5237fc0a9384260ea7da75997288f81fb99c6b0/Snowflake.Data.Tests/IntegrationTests/SFConnectionWithTomlIT.cs